### PR TITLE
Changes the readme

### PR DESCRIPTION
**Pull request by PedroPizarro**
Modifica o README visando explicar o conteúdo do repositório.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
...
+### Using docker image
+If you have Docker installed, you can use the official AsciiDoctor image:
+```
+# Pull image once
+docker pull asciidoctor/docker-asciidoctor
+# Build the HTML5 version
+docker run -v $PWD:/documents/ asciidoctor/docker-asciidoctor asciidoctor -r asciidoctor-diagram sslrules.adoc
+# Build the PDF version
+docker run -v $PWD:/documents/ asciidoctor/docker-asciidoctor asciidoctor-pdf -r asciidoctor-diagram sslrules.adoc
+```
```
Comment (fabiooju) [inline README.md:25]: ok

**Review by fabiooju — APPROVED** [review]
OK

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
-# rules-technical-challenge-el
\ No newline at end of file
+# RoboCup Brasil Small Size League Entry Level Technical Challenges
+
+This is the AsciiDoc source for the official RoboCup Small Size League Entry Level technical challenges rules.
+
+Each year has its own directory with its .adoc files containing rules for each technical challenge that will or had happened in that specific year.
```
Comment (ersaraujo) [inline README.md]: I think we can provide a brief description of the technical challenge and the impact that challenges have on the competition. @PedroPizarro

Comment (PedroPizarro) [reply README.md]: @ersaraujo Done

**Review by ersaraujo — APPROVED** [review]

**Pull request merged**